src/app/components/quick-connect-dialog/quick-connect-dialog.component.html

```diff
@@ -40,9 +40,19 @@
             </div>
         </ng-template>
         <ng-template #error>
-            <div class="error" fxLayoutAlign="center">
+            <div
+                *ngIf="!noPfs; else no_pfs"
+                class="error"
+                fxLayoutAlign="center"
+            >
                 Could not fetch any suggestions
             </div>
         </ng-template>
+        <ng-template #no_pfs>
+            <div class="error" fxLayoutAlign="center">
+                You cannot use Quick Connect because you are using the private
+                routing mode
```

Comment:
This is correct, but the relationship between routing mode and PFS configuration could change. Maybe it is safer to say something like "Quick Connect requires a pathfinding service, but no such service is configured"? This makes it less obvious what the user has to change to get QC working, though.

Reply:
Exactly, I wanted it to make it more obvious what needs to be changed. But I'll go with your way now, as someone who configures the routing mode would probably know what they do.